### DISCUSSION TITLE
Looking for the location of the files to edit to translate into others languages

### DISCUSSION THREAD

**Discussion by Mozitux** [mediacms-io/mediacms#599] [General]
**Describe the feature you'd like**
Hello, I am looking for the locations of the files that contain the words to be translated to adapt the CMS to other languages.

I was able to find some .html files containing the texts in particular for the media upload instructions, but nothing for the home page like for the "Home", "Featured", "Recommended" buttons etc.

Where can I find these types of files?

**Comment by Mozitux** [comment]
**Update:** I found the files and translated into French for now. I don't know if I have the right to share the changes, but if so, I could put the archive and the instructions as a proposal on this git as pull requests.

**Reply by RaD3v** [reply]
Hi Mozitux, where are these files located? I am trying to translate my family library, so it can be easier for my family members which does not speak English. Had some success on the html pages butbstill browsing for the rest. Thabks in afvance.

**Reply by RaD3v** [reply]
UPDATE:
Found most of the text in _commons.js, however still struggling with form_as.p from django for the login page and profile edit page. If you have any idea how to edit those will be great.

**Reply by Mozitux** [reply]
Sorry for the late response, main of translation part can be found in commons.js, other need to be found in some miscs folder.

This is actually quite difficult to translate correctly, because the files are scattered in several different folders and you have to use the grep command to find the string you want to modify.

If I have some time soon, I will suggest possible areas for improvement here and make PR.

**Reply by RaD3v** [reply]
Thank you for the reply. I found everything except the the form.as.p for the profile page and login page I have tried few solutions, but without success unfortunately. let me know if you found solution for these. Thanks.
![Screenshot 2024-05-21 175118](https://github.com/mediacms-io/mediacms/assets/36568058/8737df44-6372-4a7e-964b-0051a20923fb)

**Comment by mgogoulos** [comment]
This looks great, definitely share the work with us by creating a PR !
